org.otfried.Ipe.metainfo.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>org.otfried.Ipe</id>
+
+  <name>Ipe</name>
+  <summary>The Ipe extensible drawing editor</summary>
+
+  <url type="homepage">https://ipe.otfried.org/</url>
+  <url type="bugtracker">https://ipe.otfried.org/bugzilla.html</url>
+
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>GPL-3.0-or-later</project_license>
+
+  <description>
+    <p>
+      Ipe is a drawing editor for creating figures in PDF format. It supports making small figures for inclusion into LaTeX-documents as well as making multi-page PDF presentations.
+    </p>
+  </description>
+
+  <launchable type="desktop-id">org.otfried.Ipe.desktop</launchable>
+  <categories>
+    <category>Graphics</category>
+    <category>Qt</category>
+  </categories>
+  <content_rating type="oars-1.1"/>
+</component>
```

Comment:
missing mandatory `screenshots` and `releases`.

Reply:
Two of the historic releases don't have a date in the `news.txt` file and Ipe is not version controlled, so I can't find out when they were released. Also the 6.0 releases have spaces in the version string. Should I just omit anything older than 7.0.0 from the list?

Reply:
only the current one is required.